myst_nbs/samplers/SMC_Gaussian_Mixture_Kernel.myst.md

```diff
@@ -0,0 +1,188 @@
+---
```

Comment:
Is this a notebook you are working on? It doesn't seem to have any ipynb associated

Reply:
Got commit by mistake